include/fastboot.jinja2

```diff
@@ -18,6 +18,17 @@
 {% set pre_os_command = pre_os_command|default(true) %}
 {% set auto_login = auto_login|default(true) %}
 {% set boot_method = boot_method|default("fastboot") %}
+{% set deploy_download_timeout = deploy_download_timeout|default(target_deploy_timeout) %}
```

Comment:
This patch does two thing, please break it up.

1. fix the calculation correctly.
2. add the (pre|post)_boot_timeout

Reply:
updated